itinerant-iterators/app/obfuscate/blur_box.py

```diff
@@ -0,0 +1,38 @@
+from PIL import ImageFilter
+from PIL.Image import Image as PILImage
+from PIL.ImageFilter import Filter
+
+from .obfuscate import Obfuscate
+
+
+class BlurBox(Obfuscate):
+    """BlurBox is a form of obfuscation where it blurs all pixels in a given bounding box."""
+
+    def __init__(
+        self,
+        iterations: int = 20,
+        image_filter: Filter = ImageFilter.BoxBlur(radius=10),
+    ):
+        """Create a new blur obfuscator
+
+        Args:
+            iterations (int, optional): Number of blur iterations to apply to the bounding box. Defaults to 20.
+            image_filter (Filter, optional): Filter to apply for bluaring. Defaults to box blur blur with radius 10.
+        """
```

Comment:
```suggestion
        Args:
            iterations (int, optional): Number of blur iterations to apply to the bounding box. Defaults to 20.
            image_filter (Filter, optional): Filter to apply for bluring. Defaults to box blur blur with radius 10.
        """
```
Perhaps using [codespell](https://github.com/codespell-project/codespell) (as needed or as a pre-commit hook) would be in order :slightly_smiling_face: